core/templates/dev/head/pages/exploration_editor/translation_tab/TranslationTabDirective.js

```diff
@@ -17,22 +17,271 @@
  */
 
 oppia.directive('translationTab', [
-  'UrlInterpolationService', function(UrlInterpolationService) {
+  'UrlInterpolationService', '$timeout',
+  function(UrlInterpolationService, $timeout) {
     return {
       restrict: 'E',
       scope: {},
       link: function(scope) {
         scope.$broadcast('refreshTranslationTab');
+        $timeout( function(){
+          scope.$broadcast('openTranslationTutorial');
+        }, 5000);
```

Comment:
I didn't want to interfere with the refreshTranslationTab event so I added the delay. Another reason for that is the computer I am working on is not that good. Yes, this will prompt it every time the translator tab is activated even on refresh.

Reply:
Is it possible to target specific tests when doing e2e tests?

Reply:
I think it will be pretty annoying for translators to see translation tutor modal every time they land on the translation tab. So maybe we should record this activity and the modal should only get triggered for a new translator. (Similar to editor's tutor.) I mean to have `last_started_state_translator_tutorial` in core/storage/user/gae_models similar to `last_started_state_editor_tutorial`. What do you think?

Reply:
I agree with that design of making it a one time tutorial, so I'll go ahead and make that. 

As for the test what I mean by that, is instead of waiting for the whole test to get to the translation tab, I just want to be able just run the test with already at the translation tab. I also just learning protractor.js so if that is impossible that's fine I'll figure it out.

Reply:
Yeah, you can do it using `fdescribe` and `fit`! (you can find this inside[ run_e2e_tests.sh](https://github.com/oppia/oppia/tree/develop/scripts/run_e2e_tests.sh#L51) file.)